src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -54,24 +66,27 @@ class JesBackendCall(val backend: JesBackend,
   import JesBackend._
   import JesBackendCall._
 
-  def jesCommandLine = s"/bin/bash ${cmdInput.local} > ${stdoutJesOutput.local} 2> ${stderrJesOutput.local}"
+  def jesCommandLine = s"/bin/bash ${cmdInput.local}"
 
   val callGcsPath = backend.callGcsPath(workflowDescriptor, call.unqualifiedName, key.index)
   val callDir = GcsPath(callGcsPath)
   val gcsExecPath = GcsPath(callGcsPath + "/" + JesExecScript)
   val defaultMonitoringOutputPath = callGcsPath + "/" + JesMonitoringLogFile
 
-  private val callContext = new CallContext(callGcsPath, stdoutJesOutput.gcs, stderrJesOutput.gcs)
+  val jesStdoutFilename = jesLogStdoutFilename(workflowDescriptor, key)
+  val jesStderrFilename = jesLogStderrFilename(workflowDescriptor, key)
+  val jesStdoutGcsPath = s"$callGcsPath/$jesStdoutFilename"
+  val jesStderrGcsPath = s"$callGcsPath/$jesStderrFilename"
```

Comment:
I got a bit confused here - we use a workflowDescriptor to get a call's stdout/stderr names? And why does def stdoutStderr duplicate the calculations?

Could you add a quick comment?